x-pack/plugins/infra/public/components/asset_details/tabs/overview/metadata_summary/metadata_summary_list.tsx

```diff
@@ -78,54 +80,56 @@ const MetadataSummaryListWrapper = ({
     showTab(ContentTabIds.METADATA);
   };
 
+  const ExtraActions = () => (
+    <EuiButtonEmpty
+      data-test-subj="infraAssetDetailsMetadataShowAllButton"
+      onClick={onClick}
+      size="xs"
+      flush="both"
+      iconSide="right"
+      iconType="sortRight"
+      key="metadata-link"
+    >
+      <FormattedMessage
+        id="xpack.infra.assetDetailsEmbeddable.metadataSummary.showAllMetadataButton"
+        defaultMessage="Show all"
+      />
+    </EuiButtonEmpty>
+  );
```

Comment:
As this `ExtraActions` only returns a single button, I think you could remove it and define the button directly on the `CollapsibleSection` on line 107, WDYT?